techmatters/flex-plugins
# CHI-1200: all cases filter and sort

Primary reviewer: @stephenhand @nickhurlburt 

## Note: 
Please, ignore the commit message. It was a very silly mistake. It's supposed to be CHI-1200: all cases filter and sort

## Description

- What this pull request does.
- Bugfix: 
- Custom date filters: Shifting Dates
- Custom date filters: Custom date display
- Fix search on a category filter 


### Checklist
- [N/A] Corresponding issue has been opened
- [N/A] New tests added
- [N/A] Strings are localized
- [N/A] Tested for chat contacts
- [N/A] Tested for call contacts

### Related Issues
Fixes N/A

### Verification steps
- Note if migrations are required.
- For Custom date filters: Shifting Dates:
- Repro steps:
Select the Date Opened filter
Select Custom
Set End Date: 5/13/2022
Verify that does not Date shifts to 5/12/2022 automatically
This can be done for start date too

- For Custom date filters: Custom date display
- Replicate the above steps
- Select clear and select custom again, verify that the date fields populate for a user to fill out.

- For Fix search on a category filter 
- Search on a category filter
- Verify that the search start filtering after the user has typed three characters
<img width="1512" alt="Screenshot 2022-05-27 at 3 36 52 PM" src="https://user-images.githubusercontent.com/40429848/170729683-7f01585d-0c57-4f90-bd67-d3a46b0bb258.png">

<img width="1512" alt="Screenshot 2022-05-27 at 4 23 08 PM" src="https://user-images.githubusercontent.com/40429848/170729922-01b71255-acbc-467e-9518-d2b0884183fb.png">
